styles/styles.css

```diff
@@ -1050,9 +1086,10 @@ main .section.float-image-right picture {
 }
 
 /* stylelint-disable-next-line no-descending-specificity */
-.section.with-background-image.align-top > .section-container {
+main .section.with-background-image.align-top > .section-container, main .section.narrow.with-background-image.align-top > .section-container, main .section.narrower.with-background-image.align-top > .section-container  {
```

Comment:
The latter 2 selectors look like subsets of the first one. Is there a reason we need them explicitly?